usaspending_api/common/etl/spark.py

```diff
@@ -560,3 +564,213 @@ def create_ref_temp_views(spark: SparkSession, create_broker_views: bool = False
             spark.sql(sql_statement)
 
     logger.info(f"Created the reference views in the global_temp database")
+
+
+def load_csv_file(
```

Comment:
This function. seems a little redundant. All it seems to do is instantiate a logger, and otherwise pass the input parameters into another function. Why not instantiate the logger outside this function and use `write_csv_file()` directly?

Reply:
Actually, looks like `write_csv_file()` instantiates `logger` if it doesn't exist anyway

Reply:
This came from Keith's changes. I tried to keep his work as he put it.

Reply:
I'll remove it though. I agree with you.

Reply:
done